### DISCUSSION TITLE
Suitability of the module for a specific scenario

### DISCUSSION THREAD

**Discussion by trantor** [OpenIDC/mod_oauth2#47] [Q&A] [answered]
Hello @zandbelt .
For starters forgive the rather vague-sound question, but I find myself in a situation where I believe I might be able to use this module but I'd rather get some confirmation/advice.
Usually I provide SSO access to a specific web application via OpenID Connect using your `mod_auth_openidc` module, or via SAML through the Apache Shibboleth integration.
Now I find myself in a situation where I need to integrate with a customer which handles authentication through a custom portal and then, as far as I currently understand, provides access to my application with a JWT token containing the user information.
I am given a public key through which I can validate the token. From the token I will extract the claim containing the username which I shall pass through to my application.
From what I read I believe this module should allow me to meet those requirements, but I would like to hear your opinion.

Thanks in advance.

**Comment by zandbelt** [comment, marked as answer]
yes, that seems to be right
